app/Models/Artwork.php

```diff
@@ -126,6 +129,22 @@ public function getMiradorView()
         return $this->default_view;
     }
 
+    public function getSemanticSearchDescriptionAttribute()
+    {
+        return $this->textEmbeddingData->data['description'];
+    }
+
```

Comment:
I'd suggest moving this to the TextEmbedding class. Then calling this via `$artworkModelInstance->textEmbeddingData->semantic_search_description`